mkdocs.yml

```diff
@@ -475,7 +475,8 @@ nav:
         - data/sdks/node/index.md
         - Ampli Codegen: data/sdks/node/ampli.md
     - Other Topics:
-      - data/dynamic-configuration.md
+      - Dynamic Configuration: data/dynamic-configuration.md
+      - Next Generation SDKs: data/sdks/next-gen-sdks.md
```

Comment:
What about calling this "Analytics SDK Architecture" or "Analytics SDK Overview"?

Reply:
Also this should go above Dyn conf. 

As discussed on the call yesterday it can be moved to be directly under the "Quickstart Guide"